google/cloud/aiplatform/pipeline_jobs.py

```diff
@@ -139,6 +140,9 @@ def __init__(
             parameter_values (Dict[str, Any]):
                 Optional. The mapping from runtime parameter names to its values that
                 control the pipeline run.
+            input_artifacts (Dict[str, str]):
```

Comment:
As Sara mentioned, please add the example `{"vertex_model":"456"}` to the docstring. 

Please also specify that the full resource name(`"projects/123/locations/us-central1/metadataStores/default/artifacts/456"`) cannot be used. Because in some other SDK methods, we do allow both resource id and full resource name.

Reply:
Done.

Reply:
Can we automatically handle full artifact resource names?
We can just parse the resource names and get artifact ID.

P.S. I see that currently components may have input artifacts from different projects.
So it makes some sense to have that for pipeline input artifacts as well...

Reply:
Since we are retrieving the existing artifact from pipeline backend, I don't think cross project/location artifact is supported yet. One todo item we can do is to add validation rules either SDK and UI, or pipeline API layer to graceful handle the full resource name.

Reply:
I'm a bit confused here, could you clarify what is an artifact name?  is that the artifact resourceName such as"projects/123/locations/us-central1/metadataStores/default/artifacts/456" or the artifact display_name "some_user_specified_string" or the artifact resourceID which is the "456" portion of the resourceName?  also this mapping is already maintained in metadata system we should not ask the user to provide it separately ( with a caveat being that display_name is not unique )

Reply:
>I don't think cross project/location artifact is supported yet.

I see. Maybe I was mistaken.

Reply:
Updated the docstring after the offline sync.

Reply:
thank you! lgtm